README.md

```diff
@@ -504,7 +504,7 @@ conan build ..
 
 2. Build and install all required [dependencies](#software-requirements). 
 
-Then:
+### Building for MKL
```

Comment:
Could you please change "MKL" to "oneMKL" since that is the correct name of library?

Reply:
Updated to "oneMKL"